Containerfile

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# Copyright (C) 2023 Olliver Schinagl <oliver@schinagl.nl>
+
+ARG PYTHON_VERSION="3-alpine"
```

Comment:
I'm not a fan of alpine -- can you swap this for something like slim?

Reply:
I'm a HUGE fan of alpine :p I don't mind switching it, its your project of course, but on what grounds? Because -slim will probably bring the container from the current 75M (which is quite large imo) to whatever slim will do; i'll try though of course.

btw, for the first time (though I don't watch the log often) got this:
```
| /dev/ttyUSB0 <-- message: 2484 length: 22
| Exception in thread master thread:
| Traceback (most recent call last):
|   File "/usr/local/lib/python3.11/threading.py", line 1038, in _bootstrap_inner
|     self.run()
|   File "/usr/local/lib/python3.11/threading.py", line 975, in run
|     self._target(*self._args, **self._kwargs)
|   File "/usr/local/src/semonitor/semonitor.py", line 128, in masterCommands
|     masterGrant(state, dataFile, recFile, slaveAddr)
|   File "/usr/local/src/semonitor/semonitor.py", line 150, in masterGrant
|     masterTimer.start()
|   File "/usr/local/lib/python3.11/threading.py", line 957, in start
|     _start_new_thread(self._bootstrap, ())
| RuntimeError: can't start new thread
| /dev/ttyUSB0 --> message: 2484 length: 22
```

I wonder if that's related to alpine; as I haven't seen this before ... (and been running semonitor for 5 years now)

Reply:
> > I don't mind switching it
> 
> Thank you.
> 
> there's plenty of blog posts out there detailing experiences souring on alpine. I've had my own as well. tl;dr: alpine is often _too_ stripped down but the missing pieces are not always evident until performance or security issues pop up. if one wants to build a truly bare-bones image they should probably use distroless or scratch. the extra 75MB in this case is more than worth it

Hmm, I can only strongly dissagree here (and using alpine professionally as well. I do know however, that alpine and python aren't huge friends, mostly due to 'wheels'.

As for the 'too barebones', I guess that's to be expected when you only have busybox and musl. Actually, distroless/scratch mess up the docker-guidelines by not offering a shell. Yes, it'll save you a few k, but it makes life very difficult (even in CI). Though I ought to try klibc for that at some point, it offers an even smaller shell :)

Only thing that's not so easy with debian is virtual add, e.g. add packages, pip install, remove virtual packages (gcc and it's dependencies). So there will be some nastyness there.

Anyway, this all just gets philosophical, even though your change request is I suppose that :D  anyway, I'm changing it.

Btw, the error I posted has nothing to do with alpine, but I reduced my number of processes to 64, because, what would semonitor need more then a handful of processes for. Guess this limit somehow affects thread spawning; so lets see what it does with my new limit of 4096, if semonitor needs 'infinite' due to some leak...